src/main/java/org/fundacionjala/virtualassistant/service/SpeechService.java

```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.virtualassistant.service;
+
+import org.fundacionjala.virtualassistant.repository.ASRClient;
+import org.springframework.stereotype.Service;
+
+import java.nio.file.Path;
+
+@Service
+public class SpeechService {
+
+    private ASRClient asrClient;
+
+    public SpeechService(ASRClient asrClient) {
+        this.asrClient = asrClient;
+    }
+
+    public void sendRecord(String pathRecord) {
```

Comment:
This should return the converted record in text, this method should not have a void return type